src/components/SearchModal/CurrencyList/CurrencyList.component.tsx

```diff
@@ -168,28 +119,19 @@ function BreakLineComponent({ style }: { style: CSSProperties }) {
   )
 }
 
-export default function CurrencyList({
```

Comment:
Personally I prefer default exports since if there is a conflict in name you can rename it. I know in the contributor docs its mentioned to do exports. 

Also components with function will create a named function and its easy to track down in debugging. const create anonymous functions